app/server/versions/v1/routes/community/communityModerators.js

```diff
@@ -0,0 +1,134 @@
+const express = require("express");
+const {
+  communityAuthorization,
+} = require("../../authorization/communityAuthorization");
+const ResponseError = require("../../classes/responseError");
+const findSingleDocuments = require("../../functions/findSingleDocuments");
+
+const router = express.Router();
+
+const Community = require("../../models/community.model");
+const User = require("../../models/user.model");
+
+/**
+ * @swagger
+ * paths:
+ *  /api/community/moderators/{title}:
+ *    get:
+ *      security:
+ *        - bearerAuth: []
+ *      tags:
+ *        - Community Moderators
+ *      summary: Get all community moderators by community title or moderator count.
+ *      description: Use optional query 'count=true' to get moderators count.
+ *      parameters:
+ *        - in: path
+ *          required: true
+ *          name: title
+ *          schema:
+ *            $ref: '#/components/schemas/Community/properties/title'
+ *        - in: query
+ *          required: false
+ *          name: count
+ *          schema:
+ *            type: boolean
+ *      responses:
+ *        '404':
+ *          description: User not found. **||** <br>Community not found.
+ *        '200':
+ *          content:
+ *            application/json:
+ *              schema:
+ *                $ref: '#/components/schemas/Community/properties/moderators'
+ *        '400':
+ *          description: Bad Request.
+ */
+
+// Get all community moderators by community title
+router.get("/:title", async (req, res, next) => {
+  try {
+    req.log.addAction("Finding community.");
+    const { community } = await findSingleDocuments({
+      community: req.params.title,
+    });
+    req.log.addAction("Community found.");
+
+    req.log.addAction("Checking count query.");
+    if (req.query.count === "true")
```

Comment:
req.query.count === "true"? I don't understand how count can be "true."

Reply:
The 'query' is a value that is passed in at the end of the URL. [Examples](https://guides.emberjs.com/release/routing/query-
params/)
Because of this, queries are always strings.

The count query is a value that dictates whether or not to return an actual array of moderators or to just return a count of them.

Reply:
Awesome, thanks for the explanation.